README.md

```diff
@@ -1,2 +1,39 @@
-# scala-base-conf
-Scala Base Conference
+# Scalabase Conf 14 May 2021
+ScalaBase is a new mini-conference that will run on the 14th of May, right before this year's ScalaCon. 
```

Comment:
You have used Scalabase most other places and here you used ScalaBase

Reply:
Thank you so much for spotting this!❤️ I'll use Scalabase throughout. If we want to change it later, we always can 😊